recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        return self.soup.find("ul", {"class": "recipe-specs"}).find_all("li")
+
+    def total_time(self):
+        time = self._specs()[0].find_next("span").get_text()
+        return get_minutes(re.sub(r" \S$| ", "", time))
+
+    def category(self):
+        return normalize_string(self._specs()[1].find_next("span").get_text())
+
+    def nutrients(self):
+        nutrients = self._specs()[2].find_next("p").get_text()
+        nutrients = nutrients.split(",")
+        nutrients = [nutrient.split()[1] for nutrient in nutrients]
+        nutrients = {
+            "calories": nutrients[0] + " calories",
+            "fatContent": nutrients[1] + " grams fat",
+            "proteinContent": nutrients[2] + " grams protein",
+            "carbohydrateContent": nutrients[3] + " grams carbohydrates",
+        }
+        return nutrients
+
+    def image(self):
+        return self.schema.image()
```

Comment:
I think we should probably omit this, or provide an alternative implementation.  There doesn't appear to be any `schema.org` metadata available from this webpage (and generally this website, I think).